pennylane/operation.py

```diff
@@ -1068,6 +1068,21 @@ def _queue_category(self):
         """
         return "_ops"
 
+    def adjoint(self, do_queue=False):  # pylint:disable=no-self-use
```

Comment:
Moved to `Operator`, because nothing particularily `Operation` specific about this method.